localisation/english/russia2_localisation_l_english.yml

```
@@ -1193,64 +1193,86 @@
+ SOV_control_iran_desc:0 "Iran is a major southern security risk. It's also very useful as a potential staging point for influencing the Middle East and Indian Ocean."
+ SOV_operation_baikal_79:0 "Operation Baikal-79"
+ SOV_operation_baikal_79_desc:0 "Communist sentiments in Afghanistan are growing and may be good to have them under our control, it's one less threat to worry about."
+ SOV_influence_south_east_asia:0 "Influence South East Asia"
+ SOV_influence_south_east_asia_desc:0 "South East Asia, just getting used to their independence is ripe for communist influence."
+ SOV_friend_in_the_americas:0 "A friend in the Americas"
+ SOV_friend_in_the_americas_desc:0 "The United States has been communism's greatest enemy, they seek its containment and destruction. If we are to beat them we'll need a friend on the continent whose willing to help."
+ SOV_operation_red_eagle:0 "Operation Red Eagle"
+ SOV_operation_red_eagle_desc:0 "The time is upon us. The United States must be struck down."
+ SOV_in_the_shadows_goal:0 "In the Shadows"
+ SOV_in_the_shadows_goal_desc:0 "Gone are the days of forcing our will through direct force, now our influence extends from the cracks and shadows. Unseen but a very real threat.
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)